torch/distributed/fsdp/common_utils.py

```diff
@@ -115,3 +115,19 @@ class MixedPrecision:
     reduce_dtype: Optional[torch.dtype] = None
     buffer_dtype: Optional[torch.dtype] = None
     keep_low_precision_grads: bool = False
+
+
+@dataclass
+class CPUOffload:
+    """
+    This configures CPU offloading.
+
+    Attributes:
+        offload_params (bool): This specifies whether to offload parameters to
```

Comment:
(no need to change this PR)

Will `CPUOffload(offload_params=False)` offload anything to CPU? If not, why this is a dataclass? Do we plan to add any additional attributes here?

Reply:
> Will CPUOffload(offload_params=False) offload anything to CPU? If not, why this is a dataclass? Do we plan to add any additional attributes here?

`CPUOffload(offload_params=False)` does not offload anything to CPU; i.e. nothing changes.

As to why it is a dataclass, I think there were some plans for other settings, but I am not entirely clear. cc: @rohan-varma